Firestore/core/include/firebase/firestore/geo_point.h

```diff
@@ -32,28 +32,35 @@ namespace firestore {
  */
 class GeoPoint {
  public:
-  /**
-   * Creates a `GeoPoint` with both latitude and longitude being 0.
-   */
+  /** Creates a `GeoPoint` with both latitude and longitude set to 0. */
   GeoPoint();
 
   /**
-   * Creates a `GeoPoint` from the provided latitude and longitude degrees.
+   * Creates a `GeoPoint` from the provided latitude and longitude values.
    *
    * @param latitude The latitude as number between -90 and 90.
    * @param longitude The longitude as number between -180 and 180.
    */
   GeoPoint(double latitude, double longitude);
 
+  /** Copy constructor, `GeoPoint` is trivially copyable. */
```

Comment:
What do you think of these comments on copying/moving?

Note that `Timestamp` and `GeoPoint` are inconsistent here (`Timestamp` simply omits mentioning its copy/move operations).

Reply:
Why do we even need these? It seems like we could apply the rule of zero and avoid all of this by doing the following:

  * Assign default values in the member declarations for `latitude_` and `longitude_`
  * Remove the explicit default constructor definition
  * change the default constructor declaration to `GeoPoint() = default;`
  * Remove copy/move constructor and assignment declarations (which seem redundant).

Reply:
It's probably due to the [style guide](https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types) (emphasis mine):
> _Every class's public interface should make explicit which copy and move operations the class supports. This should usually take the form of explicitly declaring and/or deleting the appropriate operations in the public section of the declaration.
<...>
These declarations/deletions can be omitted only if they are obvious <...> a struct's copyability/movability is normally determined by the copyability/movability of its data members (__this does not apply to classes because in Google code their data members are not public__)._

However, I would argue that this class essentially _is_ a `struct`, so omitting them seems fine. Let me know what you think.

Reply:
I see. Well in that case this seems justified and the comments are fine. We should probably do the same for Timestamp then.

Reply:
Restored here and added to `Timestamp` for good measure.